sources/api/netdog/test_data/networkd/network/bond1.network

```diff
@@ -2,13 +2,9 @@
 Name=bond1
 [Link]
 RequiredForOnline=true
-RequiredFamilyForOnline=both
 [Network]
 ConfigureWithoutCarrier=true
-DHCP=yes
+DHCP=ipv4
```

Comment:
These changes correct a typo I made in the config in an earlier commit.  This bond is meant to only have DHCP4.